source/user-manual/reference/ossec-conf/github-module.rst

```diff
@@ -0,0 +1,227 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+.. _github-module:
+
+github
+=======
+
+.. versionadded:: 4.3.0
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<github>
+		</github>
+
+Configuration options of the GitHub module.
+
+
+Options
+-------
+
+- `enabled`_
+- `run_on_start`_
+- `only_future_events`_
+- `interval`_
+- `time_delay`_
+- `api_auth`_
+- `api_auth\\org_name`_
+- `api_auth\\api_token`_
+- `api_parameters\\event_type`_
+
+
++----------------------------------------+----------------------------------------------+
+| Options                                | Allowed values                               |
++========================================+==============================================+
+| `enabled`_                             | yes, no                                      |
++----------------------------------------+----------------------------------------------+
+| `run_on_start`_                        | yes, no                                      |
++----------------------------------------+----------------------------------------------+
+| `only_future_events`_                  | yes, no                                      |
++----------------------------------------+----------------------------------------------+
+| `interval`_                            | A positive number + suffix                   |
++----------------------------------------+----------------------------------------------+
+| `time_delay`_                          | A positive number + suffix                   |
++----------------------------------------+----------------------------------------------+
+| `api_auth`_                            | N/A                                          |
++----------------------------------------+----------------------------------------------+
+| `api_auth\\org_name`_                  | Any string                                   |
++----------------------------------------+----------------------------------------------+
+| `api_auth\\api_token`_                 | Any string                                   |
++----------------------------------------+----------------------------------------------+
+| `api_parameters`_                      | N/A                                          |
++----------------------------------------+----------------------------------------------+
+| `api_parameters\\event_type`_          | web, git, all                                |
++----------------------------------------+----------------------------------------------+
+
+enabled
+^^^^^^^
+
+Enabled the GitHub wodle.
+
++--------------------+-----------------------------+
+| **Default value**  | no                          |
```

Comment:
This was changes to **yes** to be the same as the other modules. Please, change it.